src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/AppContextDefaultValues.cs

```diff
@@ -2,68 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Since multiple assemblies (WindowsBase, PresentationFramework, PresentationCore, PresentationBuildTasks)
+// share this file/class and have InternalsVisibleTo set between them, there will be a compiler warning
+// that the type is found both in the source and in a referenced assembly. The compiler will prefer 
+// the version of the type defined in the source once the warning is supressed and won't compile the rest.
+
+// In order to disable the warning for this type we are disabling the warning for this entire file.
+#pragma warning disable CS0436 // Type conflicts with imported type
```

Comment:
@h3xds1n, Imo the more granular we could go with respect to warning suppressions the better it is. Can we not explore disabling the warning at class level or `PopulateDefaultValuesPartial` method itself?

Reply:
@harshit7962 I'll check later today whether we could constrain it better but at least class-level will be still required iirc.

Reply:
@harshit7962 I see why it didn't work previously for me; I didn't include the cleanup of the warnings in the switch-specific files. I've fixed it in this file and also constrained it in `LocalAppContext` to suppress just in the static constructor; should do the job better.

I've unified the comment where the suppression is actually needed.

Thanks as always for the great feedback!

Reply:
Also, just thinking, `LocalAppContext` should not be `partial` but `static`.

There seems to be no reason for this to be an instance class as all the members are static and it is not using any partial stuff either, each library gets a full compiled copy of it. I'd push a change for this small thing if you agree with me.

Reply:
@h3xds1nz, seems fair. We can go ahead and have the class as static and non-partial. Resolving this conversation as of now.

Reply:
@harshit7962 Pushed the change, you can re-approve.